src/components/dashboard/PendingTxs/PendingTxListItem.tsx

```diff
@@ -19,6 +23,10 @@ type PendingTxType = {
 const PendingTx = ({ transaction }: PendingTxType): ReactElement => {
   const router = useRouter()
   const { id } = transaction
+  const { safe } = useSafeInfo()
+  const wallet = useWallet()
+  const signable = wallet ? isSignableBy(transaction, wallet.address) : false
+  const executable = wallet ? isExecutable(transaction, wallet?.address, safe) : false
```

Comment:
I would suggest to keep the naming convention of the other places we use these functions: `canSign`, `canExecute`